convert2rhel/utils.py

```diff
@@ -471,10 +471,17 @@ def download_pkg(
 
     output, ret_code = run_cmd_in_pty(cmd, print_output=False)
     if ret_code != 0:
-        loggerinst.warning(
-            "Couldn't download the %s package using yumdownloader.\n"
-            "Output from the yumdownloader call:\n%s" % (pkg, output)
-        )
+        loggerinst.warning("you are using an env that overides an inhibitor and 
```

Comment:
The `you` at the beginning of the message should be capitalized.

Reply:
the new code has changed the warning message and doesn't use this message anymore